CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Unreleased
 
+Added:
+
+- Configuration option `new_buffer.input_visibility` to control input field visibility: always shown or following the focused buffer.
```

Comment:
I feel like we should rename this config option to just `buffer` at this point. Maybe with serde alias to `new_buffer` for backwards compatibility.

Now that we don't have the settings vs config confusion, I'm not sure `new_buffer` here makes sense as the name.

Reply:
Agree!